src/test/java/com/epam/steps/admin/AdminStepsTeacherSection.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.steps.admin;
+
+import com.epam.pages.main.AdminPage;
+import com.epam.pages.popup.CreatePopup;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AdminStepsTeacherSection {
```

Comment:
Please, can you change the name to CreateTeacherSteps, because in the super_admin directory, there is a class with the name of CreateAdminSteps, and they functional is similar, different is only that one used for admin creation, and the another one is for teacher creation

Reply:
Done, Davit jan